src/System.Runtime.WindowsRuntime/tests/Resources/System.Runtime.WindowsRuntime.Tests.rd.xml

```diff
@@ -0,0 +1,6 @@
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
```

Comment:
As per comment: https://github.com/dotnet/corefx/pull/21658#issuecomment-311869642

You might want to log an issue to investigate this and add it to this file.